guides/DomainEvents.md

```diff
@@ -0,0 +1,56 @@
+# Webhook structure
+
+Wix exposes webhooks for standard create, update, and delete actions, and relevant additional actions.
+Most webhooks are organized in a standard format, as described below. Wix also has some legacy webhooks that don't comply with the standardized format.
+
+The data payload will include the following as an encoded JWT:
+
+| Field name | Description | 
+| :-------------- | :------- |  
+| `data` | |
+| `data.eventType`| Webhook event type, as documented in each webhook. For example: `wix.contacts.v4.contact_merged`. |
+| `data.instanceId`| App instance ID, the identifier of the site where the event occurred. |
+| `data.data`| Data payload as stringified JSON. See the **Data payload** table. |
+| `data.identity` | Identity data as stringified JSON. See the **Identity payload** table. | // Identity of what??
+
+## Data payload
+The `data.data` payload will always include the following fields:
```

Comment:
```suggestion
The `data.data` payload includes the following fields:
```

Reply:
@laurakedem  I wanted to differentiate between the fields that are always returned, and the ones that change based on the webhook type. If you think that differentiation isn't necessary, I'll cut it